e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -555,6 +555,36 @@ test.describe('Example Imagery in Time Strip', () => {
     // Navigate to timestrip
     await page.goto(timeStripObject.url);
   });
+
+  test.describe('Example Imagery in Flexible layout', () => {
+    let flexibleLayout;
+    test.beforeEach(async ({ page }) => {
+      await page.goto('./', { waitUntil: 'domcontentloaded' });
+
+      flexibleLayout = await createDomainObjectWithDefaults(page, { type: 'Flexible Layout' });
+
+      // Create Example Imagery inside the Flexible Layout
+      await createDomainObjectWithDefaults(page, {
+        type: 'Example Imagery',
+        parent: flexibleLayout.uuid
+      });
+
+      // Navigate back to Flexible Layout
+      await page.goto(flexibleLayout.url);
+    });
+
+    test('Can double-click on the image to view large image', async ({ page }) => {
+      // Double-click on the image to open large view
+      await page.locator('.image-wrapper').dblclick();
+
+      // Check if the large view is visible
+      await page.waitForSelector('.c-imagery__main-image__bg', { state: 'visible' });
+
+      // Close the large view
+      await page.locator('[aria-label="Close"]').click();
```

Comment:
We prefer to avoid `.locator()` if at all possible. This could be:
```suggestion
      await page.getByLabel('Close').click();
```